nixos/modules/system/boot/loader/systemd-boot/systemd-boot.nix

```diff
@@ -238,6 +245,17 @@ in {
       '';
     };
 
+    mirroredBoots = lib.mkOption {
+      type = lib.types.listOf lib.types.str;
```

Comment:
`grub.nix`'s [mirroredBoots' type](https://github.com/NixOS/nixpkgs/blob/ec6cc9c826e08ec1aa4212c40ad92eb5186a9053/nixos/modules/system/boot/loader/grub/grub.nix#L235) is more sophisticated, maybe it would be more correct to use the same or a subset.